releasedir/provider.go

```diff
@@ -103,6 +103,8 @@ func (p Provider) newBlobstore(dirPath string) boshblob.DigestBlobstore {
 		blobstore = boshblob.NewLocalBlobstore(p.fs, p.uuidGen, options)
 	case "s3":
 		blobstore = NewS3Blobstore(p.fs, p.uuidGen, options)
```

Comment:
Yea, I like it better the way you did it. Thanks for answering my question though :) I'm guessing it'll come up on the main repo, so that's a good idea to mention it in the commit message.